src/viper/common.ml

```diff
@@ -3,4 +3,3 @@ exception Unsupported of Source.region * string
 
 let unsupported at sexp =
   raise (Unsupported (at, (Wasm.Sexpr.to_string 80 sexp)))
-
```

Comment:
Is it intended

Reply:
Nope, perhaps I left it when I was stripping `zip` family. This file already has a trailing newline, I can restore the removed one if you want.